mesonbuild/minstall.py

```diff
@@ -533,7 +563,7 @@ def install_targets(self, d):
             if file_copied:
                 self.did_install_something = True
                 try:
-                    depfixer.fix_rpath(outname, t.rpath_dirs_to_remove, install_rpath, final_path,
+                    depfixer.fix_rpath(outname, sorted(t.rpath_dirs_to_remove), install_rpath, final_path,
```

Comment:
Just curious but why add `sorted` here`?

Reply:
`rpath_dirs_to_remove` is actually a set, while `fix_rpath` expects a list, and I wasn't sure if order mattered. But looking at it more it looks like this value is only used for a single `if x in container` check, so just passing a set would be better. I'll fix that.